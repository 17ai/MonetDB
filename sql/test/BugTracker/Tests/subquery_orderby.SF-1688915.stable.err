stderr of test 'subquery_orderby.SF-1688915` in directory 'sql/test/BugTracker` itself:


# 09:40:47 >  
# 09:40:47 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38406" "--set" "mapi_usock=/var/tmp/mtest-20871/.s.monetdb.38406" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/mk/mosaic//Linux/var/MonetDB/mTests_sql_test_BugTracker"
# 09:40:47 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/mosaic//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 38406
# cmdline opt 	mapi_usock = /var/tmp/mtest-20871/.s.monetdb.38406
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/mosaic//Linux/var/MonetDB/mTests_sql_test_BugTracker
# cmdline opt 	gdk_debug = 536870922

# 09:40:47 >  
# 09:40:47 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-20871" "--port=38406"
# 09:40:47 >  


# 09:40:47 >  
# 09:40:47 >  "Done."
# 09:40:47 >  

