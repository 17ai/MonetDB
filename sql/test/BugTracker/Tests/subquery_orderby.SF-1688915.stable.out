stdout of test 'subquery_orderby.SF-1688915` in directory 'sql/test/BugTracker` itself:


# 12:35:21 >  
# 12:35:21 >   mserver5 "--config=/ufs/niels/scratch/monetdb/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set "monet_mod_path=/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=31630 --set xrpc_port=49305 --set monet_prompt= --trace "--dbname=mTests_src_test_BugTracker" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 12:35:21 >  

# MonetDB server v5.3.0, based on kernel v1.21.0
# Serving database 'mTests_src_test_BugTracker'
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-2007 CWI, all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://koala.ins.cwi.nl:31630/
# MonetDB/SQL module v2.21.0 loaded

Ready.
#function user.main():void;
#    clients.quit();
#end main;


# 12:35:21 >  
# 12:35:21 >  mclient -lsql -umonetdb -Pmonetdb --host=koala --port=31630 
# 12:35:21 >  


# 09:40:47 >  
# 09:40:47 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-20871" "--port=38406"
# 09:40:47 >  

#select name from (select name, id from tables order by id) as x;
% .x # table_name
% name # name
% varchar # type
% 17 # length
[ "schemas"	]
[ "types"	]
[ "functions"	]
[ "args"	]
[ "sequences"	]
[ "dependencies"	]
[ "connections"	]
[ "_tables"	]
[ "_columns"	]
[ "keys"	]
[ "idxs"	]
[ "triggers"	]
[ "objects"	]
[ "_tables"	]
[ "_columns"	]
[ "keys"	]
[ "idxs"	]
[ "triggers"	]
[ "objects"	]
[ "tables"	]
[ "columns"	]
[ "db_user_info"	]
[ "users"	]
[ "user_role"	]
[ "auths"	]
[ "privileges"	]
[ "querylog_catalog"	]
[ "querylog_calls"	]
[ "querylog_history"	]
[ "tracelog"	]
[ "sessions"	]
[ "optimizers"	]
[ "environment"	]
[ "queue"	]
[ "rejects"	]
[ "keywords"	]
[ "table_types"	]
[ "dependency_types"	]
[ "netcdf_files"	]
[ "netcdf_dims"	]
[ "netcdf_vars"	]
[ "netcdf_vardim"	]
[ "netcdf_attrs"	]
[ "storage"	]
[ "storagemodelinput"	]
[ "storagemodel"	]
[ "tablestoragemodel"	]
[ "statistics"	]
[ "files"	]
[ "sq"	]
[ "rg"	]
[ "pg"	]
[ "export"	]
[ "systemfunctions"	]

# 09:40:47 >  
# 09:40:47 >  "Done."
# 09:40:47 >  

